code/client/src/pages/Show/About.jsx

```diff
@@ -144,9 +145,12 @@ const About = ({ address }) => {
           </Col>
         </TallRow>}
       <Row style={{ marginTop: 24 }}>
-        <Popconfirm title='Are you sure？' onConfirm={() => deleteWalletLocally({ wallet, wallets, dispatch, history })}>
-          <Button type='primary' shape='round' danger size='large' icon={<DeleteOutlined />}>Delete locally</Button>
-        </Popconfirm>
+        <Space>
```

Comment:
Let's have the items spread out (flex space-between), and delete action on the left, so it is consistent with overall style. (negative action on the left, positive on the right)